config/envs/rhdm-authoring-ha.yaml

```diff
@@ -383,7 +420,7 @@ others:
                           "value": "8888"
                         }
                       ],
-                      "image": "amq-broker-72-openshift:1.3",
+                      "image": "registry.redhat.io/amq-broker-7/amq-broker-73-openshift:1.3",
```

Comment:
hmm, i expect this will be problematic for users. this will likely require a user to setup their pam project for authenticated registry pulls... rather than allowing the DC/stateful set to first check for an imagestream in openshift project.